modules/base/tests/playwright/ServerRoutes.spec.js

```diff
@@ -55,3 +55,43 @@ test('request on different path doesnt invoke server route', async ({ page }) =>
     },
   )
 })
+
+test('server route with path prefix', async ({ page }) => {
+  await makeProject(
+    {
+      modules: [
+        '@storefront-x/base',
+        '@storefront-x/vue',
+        '@storefront-x/vue-i18n',
+        [
+          'my-module',
+          {
+            config: {
+              'VUE_I18N_LOCALES.ts': `export default [
+                  {
+                    name: 'sk',
+                    locale: 'sk-SK',
+                    prefix: '/sk',
+                  },
+                ]
+                `,
+            },
+            server: {
+              routes: {
+                '[prefix]': {
+                  'hello.js': `export default (req, res) => res.send(req.baseUrl)`,
+                },
+              },
+            },
+          },
+        ],
+      ],
+    },
+    async ({ url }) => {
+      await page.goto(url + '/sk/hello', { waitUntil: 'networkidle' })
+      await expect(await page.locator('body')).toHaveText('/sk/hello')
+      await page.goto(url + '/hello', { waitUntil: 'networkidle' })
+      await expect(await page.locator('body')).toHaveText('/hello')
```

Comment:
This should not work because it has no prefix so it should not match any available route. There is no "just /hello" route.